src/components/FuncParams.js

```diff
@@ -0,0 +1,47 @@
+import React, {Component} from 'react';
+import {TextField, CardText} from 'material-ui';
+
+class FuncParams extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {params: props.params}
```

Comment:
Same thing with state here, `props.params` already has the value.  So you dont need this line, just use this.props.params in the render (but you'll want to destructure it after render() to `const {params} = this.props`